drafts/25-xxx-specifications.txt

```diff
@@ -206,64 +206,348 @@ to11. The following tokens also appear in the illustrative syntax below.
+
+
+3.4.1 Static semantics specifications
+--------------------------------------
+
+in02. In the first form, the CHARACTER_STRING may be delimited with
+      single-quote "'" or double-quote '"' characters.
+
+in04. A "_kind" specifier shall not appear following the CHARACTER_STRING.
+
+in06. In the second form, the character-list is any sequence of
+      processor-specific characters.
+
+
+3.4.2 Evaluation semantics specifications
+-----------------------------------------
+
+in20. The preprocessor searches in processor-defined places for the
+      file denoted by the CHARACTER_STRING or the character-list.
+
+in22. It is an error if the preprocessor can't locate the specified file.
 
-3.3 The '#if', '#ifdef', '#ifndef', '#elif', '#elifdef',
+in24. If the file is located, the preprocessor replaces the #include
+      directive line with the contents of that file.
+
+in26. A #include directive may appear in an included file, up to a
+      processor-defined nesting limit.
+
+
+3.5 The '#if', '#ifdef', '#ifndef', '#elif', '#elifdef',
       '#elifndef', '#else', '#endif' directives
 ---------------------------------------------------------
 
+Example syntax:
+
+
+
+3.5.1 Static semantics specifications
+--------------------------------------
+
 
-3.4 The '#error' and '#warning' directives
+3.5.2 Evaluation semantics specifications
+-----------------------------------------
+
+
+3.6 The '#error' and '#warning' directives
 ------------------------------------------
 
+Example syntax:
+        #error token-list EOL
+        #warning token-list EOL
+
+The #error and #warning directives are used to announce user-defined
+error conditions in the input.
+
+
+3.6.1 Static semantics specifications
+--------------------------------------
+
+
+3.6.2 Evaluation semantics specifications
+-----------------------------------------
 
-3.5 The '#line' directive
+ew02. The token-list is expanded.
+
+ew04. The preprocessor produces a diagnostic message that includes
+      the result of the token expansion.
+
+ew06. The #error directive causes the compilation process to and after
+      preprocessing the input files.
+
+ew08. The #warning directive allows compilation to continue after
+      preprocessing.
+
+ew10. Additional behavior is processor-specific.
+
+
+3.7 The '#line' directive
 -------------------------
 
+Example syntax:
+        #line WHOLE_NUMBER EOL
+        #line WHOLE_NUMBER CHARACTER_STRING EOL
+
+
+3.7.1 Static semantics specifications
+--------------------------------------
+
+li02. The WHOLE_NUMBER shall not be zero.
+
+li04. The WHOLE_NUMBER shall not exceed 2147483647.
+
+
+3.7.2 Evaluation semantics specifications
+-----------------------------------------
+
+li20. In the first example, the #line directive causes the preprocessor
+      to behave as if the following sequence of Fortran source lines or
+      comment lines begins with a source line that has a line number
+      specified by the WHOLE_NUMBER.
```

Comment:
This wording seems to imply the altered numbering does not apply to directive lines, which it also does.

We've also switched from calling the syntax "forms" to "examples". Use "forms"

Rewording for more precision:
```suggestion
li20. In both forms, the #line directive causes the processor
      to behave as if the following sequence of lines begins with a 
      presumed line number specified by the WHOLE_NUMBER.
```